### PR TITLE
Change ordering of templates returned to Dashboard to orderBy code

### PR DESCRIPTION
**Pull request by nmadruga**
Quick fix to try sort out the problem found in https://github.com/openmsupply/conforma-templates/issues/19

### CODE REVIEW

**Review by CarlosNZ — APPROVED** [review]
Awesome -- quick and easy solution 👍

Just one note -- the order of the categories is being affected too. Which ever template comes first, its category is first displayed. Is this okay, or do we want to have categories display in order by their code, too?

File: src/utils/generated/graphql.tsx [hunk 1]
```diff
@@ -61,6 +61,36 @@ export type Query = Node & {
   counters?: Maybe<CountersConnection>;
   /** Reads and enables pagination through a set of `DataTable`. */
   dataTables?: Maybe<DataTablesConnection>;
+  /** Reads and enables pagination through a set of `DataTableActiveIngredient`. */
```
Comment (CarlosNZ) [inline src/utils/generated/graphql.tsx:64]: Don't want to commit the Angola types to repo.

Comment (nmadruga) [reply src/utils/generated/graphql.tsx:64]: Ok - cool I'll revert this change by using core_templates loaded in

**Comment by nmadruga** [comment]
> Just one note -- the order of the categories is being affected too. Which ever template comes first, its category is first displayed. Is this okay, or do we want to have categories display in order by their code, too?

Probably good to have categories sorted by code as well for consistency. Will work on it now

**Comment by nmadruga** [comment]
Done!

**Pull request merged**